cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -435,6 +435,31 @@ spec:
     singular: cephblockpool
   scope: Namespaced
   version: v1
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            failureDomain:
+                type: string
+            replicated:
+              properties:
+                size:
+                  type: integer
+                  minimum: 0
+                  maximum: 9
+                targetSizeRatio:
+                  type: number
+            erasureCoded:
+              properties:
+                dataChunks:
+                  type: integer
+                  minimum: 1
```

Comment:
Looks like we need the minimum for the EC settings to be 0 as well. I'm seeing an error in the operator log. The pools are successfully created, but the status update is causing this error:
```
2020-02-05 23:32:31.300868 E | op-pool: Unable to update the cephBlockPool replicapool1 status CephBlockPool.ceph.rook.io "replicapool1" is invalid: [spec.erasureCoded.dataChunks: Invalid value: 1: spec.erasureCoded.dataChunks in body should be greater than or equal to 1, spec.erasureCoded.codingChunks: Invalid value: 1: spec.erasureCoded.codingChunks in body should be greater than or equal to 1]
```